### PR TITLE
Home and about pages

### PR DESCRIPTION
**Pull request by AKeeganDev**
Here is the pull request for my Capstone project. A video tour of the website can be found here:
https://www.loom.com/share/faf3fc4daa44496ca58561015dafe4be

This pull request features a completely responsive and complete Home & About page for the fake business site "Aaron Keegan "dot" Dev.

Styling is done mostly using flexbox and Javascript. I opted not to use any frameworks so every pixel is designed by completely by hand and entirely through my own code.

Some assets were borrowed directly from Cindy's design like her backgrounds, icons and images for my Testimonial section. I had removed their names and provided a fake generic name and quote for each.

Thank you!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,12 +5,115 @@
...
+                </li>
+                <li class="contact"><a href="#">Contact</a></li>
+            </ul>
+        </div>
+    </nav>
+    <div class="sidenav overflow-hidden">
+        <a href="./pages/about.html">About</a>
+        <a href="#">Services</a>
+        <a href="#">Clients</a>
+        <a href="#">Contact</a>
```
Comment (thecodechaser) [inline index.html]: - [x] Please wrap these links in a list, You can use `ul` list, When we have similar elements it's good practice to wrap them in a list and it also makes sure that our code is semantic.

File: pages/about.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        </li>
+        <li class="contact"><a href="#">Contact</a></li>
+      </ul>
+    </div>
+  </nav>
+  <div class="sidenav overflow-hidden">
+    <a href="../index.html">Home</a>
+    <a href="#">Services</a>
+    <a href="#">Clients</a>
+    <a href="#">Contact</a>
```
Comment (thecodechaser) [inline pages/about.html]: - [x] Please wrap these links in a list, You can use `ul` list, When we have similar elements it's good practice to wrap them in a list and it also makes sure that our code is semantic.

File: index.html [hunk 3]
```diff
@@ -5,12 +5,115 @@
...
+            </li>
+            <li>
+                Language
+            </li>
+            <li></li>
+        </ul>
+    </div>
+    <nav class="navbar">
+        <div class="nav-container">
+            <div id="logo" class="logo"></div>
```
Comment (thecodechaser) [inline index.html]: - [x] Please use anchor tag `<a>` for your logo, Logos are supposed to be links.

File: css/style.css [hunk 4]
```diff
@@ -1,9 +1,758 @@
...
+}
+
+.welcome-mobile-background {
+  background-image: url(../img/mobile_bg.png);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+@media screen and (min-width: 752px) {
+  .navbar {
```
Comment (thecodechaser) [inline css/style.css]: - [x] The media queries to apply for the desktop screen should be 768px. Please make sure your styles for desktop screens should start from 768px.

Please see the image below:

image

File: css/style.scss [hunk 5]
```diff
@@ -1,9 +1,640 @@
...
+
+.welcome-mobile-background {
+  background-image: url(../img/mobile_bg.png);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+// Fixes top-most nav to stay close to alignment with nav-bar
+@media screen and (min-width: 752px) {
+  .navbar {
```
Comment (thecodechaser) [inline css/style.scss]: - [x] The media queries to apply for the desktop screen should be 768px. Please make sure your styles for desktop screens should start from 768px.

Please see the image below:

image

File: index.html [hunk 6]
```diff
@@ -5,12 +5,115 @@
...
+            <div class="project-card">
+                <div class="project-icon process"></div>
+                <div class="title">My Process</div>
+                <div class="description">Click here to learn about my design process</div>
+            </div>
+            <div class="project-card">
+                <div class="project-icon faq"></div>
+                <div class="title">FAQ</div>
+                <div class="description">Click here see my FAQ (frequently asked questions)</div>
+            </div>
```
Comment (thecodechaser) [inline index.html:106]: - [x] Please apply hover effects to your cards, So when we hover over any card it should display a white border around the card

See the below image for reference 

image

File: index.html (hunk 1 above)
Comment (thecodechaser) [inline index.html]: - [x] Great job on the navbar menu but there is a little issue. you are supposed to stick to the design as much as possible and your webpage should look identical to the Behance template. You are missing the CC campaign from the menu. Please add that button and make sure it follows the design on the template.

See the image for reference

image

File: index.html [hunk 7]
```diff
@@ -5,12 +5,115 @@
...
+        </div>
+    </section>
+    <section id="testimonials">
+        <h2>Client Testimonials</h2>
+        <div class="divider"></div>
+        <div id="container" class="testimonial-container"></div>
+    </section>
+    <footer class="footer">
+        <div class="logo"></div>
+        <div class="copyright">
```
Comment (thecodechaser) [inline index.html]: - [x] Your design is missing the partner's section before the footer, Please make sure your homepage has the partner section and it should follow the design on the template.

See the image for reference 

image

Comment (thecodechaser) [inline index.html]: - [x] Please make sure your footer has a white background color as on the design

See the image for reference

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED 🔴🟡 🔴 

## Hi @AKeeganDev 👋 

### You are doing a marvelous job on the capstone project and this is one of the best projects I have seen so far!
I really like your presentation for the project 👍

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good accessibility report. ✔️

## Required Changes ♻️
- [x]  In the presentation video you should show some piece of code that you are very proud of, Please make sure your presentation video includes that.
- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x]  Please apply the hover effect to your navbar menu links.
- [ ] Please add some screenshots of the project in the readme file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: pages/about.html [hunk 8]
```diff
@@ -104,7 +106,9 @@ <h3>Is Web-Dev Right For Me?</h3>
     </div>
   </section>
   <footer class="footer">
-    <div class="logo"></div>
+    <a href="../index.html">
```
Comment (Cathella) [inline pages/about.html:109]: - [ ] [OPTIONAL] The file path to the logo can be a `#` or just `index.html` to reference its self

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @AKeeganDev 👋🏻, 

A beautiful job you have done on your capstone project ❤️. 
Also good job on making the requested changes 💪🏻!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**